apps/app/src/Screens/Friends.tsx

```diff
@@ -0,0 +1,17 @@
+import { Menu } from "@/components/Menu/Menu"
+import { View, Text } from "react-native"
+
+import type { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { RootStackParamList } from "./Types/Stack";
+
+type FriendsProps = NativeStackScreenProps<RootStackParamList, "Friends">
+
+export const Friends = ({navigation, route}: FriendsProps) => {
+  return (
+    <View className="flex-1 items-center justify-center bg-orange-200 space-y-10">
+      <Text>This is {route.params.name}'s friends</Text>
+      <Menu navigation={navigation} route={route} />
```

Comment:
**question (non-blocking):** You're including this menu on every screen, is there a way to do that without copy-pasting it to every screen?

Reply:
As we talked, I will do later to do not block this PR for so long 😉